app/store/popup_store.ts

```diff
@@ -1,6 +1,9 @@
-import { createStore, applyMiddleware, compose } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
 import { createLogger } from 'redux-logger'
 import { Middleware } from 'react-redux/node_modules/redux';
+import thunkMiddleware from 'redux-thunk';
+import aliases from './aliases'
+import { alias } from 'react-chrome-redux';
```

Comment:
what does this do?

Reply:
Refer [this](https://github.com/tshaddix/react-chrome-redux/wiki/Advanced-Usage#alias)

Reply:
Okay, more precisely, why do you want to use this?

Reply:
nevermind, I think I understand.